lib/bpk-component-text/src/BpkText-test.common.js

```diff
@@ -57,6 +57,34 @@ const commonTests = () => {
       });
     });
 
+    it('should render correctly when textStyle is set to inherit', () => {
+      const tree = renderer
+        .create(
+          <BpkText textStyle="xxxl">
+            <BpkText textStyle="inherit">Some bold text</BpkText>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
+            commodo ligula eget dolor. Aenean massa. Cum sociis natoque
+            penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+          </BpkText>,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly when weight is set to inherit', () => {
+      const tree = renderer
+        .create(
+          <BpkText textStyle="xxxl">
+            <BpkText weight="emphasized">Some bold text</BpkText>
```

Comment:
This should be weight="inherit" and have a weight set on the parent.

Reply:
Changed